Make-a-pull-request.md

```diff
@@ -92,7 +92,9 @@ You can change your most recent commit message using `git commit --amend`. **It
 
 Once your feature is ready, you can open a pull request (PR)!
 
-Go to your fork on GitHub, select your branch from the dropdown menu, and click "pull request". Ensure that the base repository is `oppia/oppia` and that the base branch is `develop`. The head repository should be your fork, and the head branch should be your branch. If you don't see the repository, click the link to `compare across forks`. On this page, you can also see your commits and your changes.
+Go to your fork on GitHub, select your branch from the dropdown menu, and click "pull request". Ensure that the base repository is oppia/oppia and that the base branch is develop. The head repository should be your fork, and the head branch should be your branch. If you don't see the repository, click the link to compare across forks. On this page, you can also see your commits and your changes.
+
+### Detailed Instructions
 
 ![The GitHub PR creation screen](images/makingAPR.png)
```

Comment:
The paragraph above describes this image, so the "Detailed Instructions" header should go above the preceding paragraph and just under "Once your feature is ready, ..."

Also, what about "How to make a pull request" instead of "Detailed Instructions" to be a little more descriptive?

Reply:
Thanks, I'll stick to your guidelines.